packages/portfolio-contract/src/pos.exo.ts

```diff
@@ -66,10 +67,12 @@ export const preparePosition = (
   zone: Zone,
   emptyTransferState: TransferStatus,
   publishStatus: PublishStatusFn,
-) =>
-  zone.exoClass(
+) => {
+  const PositionI = M.interface('Position', {}, { defaultGuards: 'raw' });
```

Comment:
With `{}, { defaultGuards: 'raw' }`, you actually made the checking weaker than the original. In the original, the `undefined` in the interface guard position is equivalent to `{}, { defaultGuards: 'passable' }`.

Please change to
```suggestion
  const PositionI = M.interface('Position', {}, { defaultGuards: 'passable' });
```
which is a pure refactor equivalent to the original, but a starting point for filling in method guards and eventually removing the `defaultGuards`.

Reply:
@copilot please change it as above

@erights the bot only seems to act when directly addressed